sources/packages/backend/libs/test-utils/src/factories/application-exception-request.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  ApplicationException,
+  ApplicationExceptionRequest,
+  User,
+} from "@sims/sims-db";
+import * as faker from "faker";
+
+export function createFakeApplicationExceptionRequest(
```

Comment:
@ann-aot @dheepak-aot @guru-aot @sh16011993 I got some comments some reviews ago about comments on this type of factory. I was not looking forward to adding comments but since that PR I got the idea that there was a team desire to have these methods also commented. Are we on the same page that we need to comment also these methods?

Reply:
That is what I heard during my PR and I am in same page to follow adding comments for these methods.